netlify.toml

```diff
@@ -0,0 +1,19 @@
+# “command” is your build command.
+# “publish” is the directory to publish (relative to the root of your repo).
```

Comment:
@nobioma1  any idea why am I seeing the whole file in didd

Reply:
@samarv The changes are from the "Setup Add App to Slack button" which @ropeks pulled into this branch before creating the view for single shoutouts

Reply:
@nobioma1  is it the same as the one in `develop`. do we need to scrap this file?

Reply:
@samarv the `netlify.toml` issue has been fixed by pulling the updated `develop` branch and merging it to this branch by @ropeks